src/main/java/at/hannibal2/skyhanni/config/features/Garden.java

```diff
@@ -707,7 +707,7 @@ public static class CustomSpeed {
     public boolean nextJacobContestWarn = false;
 
     @Expose
-    @ConfigOption(name = "Warning Time", desc = "Set the warning time in seconds before the Jacobs contest begins.")
+    @ConfigOption(name = "Warning Time", desc = "Set the warning time in seconds before a jacob contest begins.")
```

Comment:
If we are being pedantic, this should be a "Jacob's farming contest" or "Jacob's contest"

Reply:
Fixed with newest commit